docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory to `/app` can introduce unwanted files into the container. It's better to be selective with what you mount to avoid potential conflicts or security issues.